packages/postcss-short-css-vars/index.test.js

```diff
@@ -68,12 +68,21 @@ describe('postcss-short-css-vars', () => {
   });
 
   describe('ignore', () => {
+    it('ignores variable names via string', () => {
+      const input = ':root{--custom-var-one: red; --custom-var-two: 1.2rem}' +
+        '.my-class{ color: var(--custom-var-one, var(--custom-var-two))}';
+      const expected = ':root{--custom-var-one: red; --1tn4ykf: 1.2rem}' +
+        '.my-class{ color: var(--custom-var-one, var(--1tn4ykf))}';
+      const result = run(input, { ignore: 'one$' });
```

Comment:
since you have `$` in there, maybe show a input->output where `one` exists but not at the end of the string

Reply:
same for the rule below.

Reply:
I only added the $ to demonstrate it does in fact get used as regexp. I'll adjust this to a similar pattern suggested in your [other comment](https://github.com/godaddy/short-css-vars/pull/1#discussion_r481321093).